projects/components/src/menu-dropdown/menu-dropdown.component.scss

```diff
@@ -1,4 +1,5 @@
 @import 'interaction';
```

Comment:
Please remove this. `mixins` covers this

Reply:
Okay.